Tests/SotoCoreTests/TimeStampTests.swift

```diff
@@ -53,7 +53,7 @@ class TimeStampTests: XCTestCase {
         let a = A(date: Date(timeIntervalSince1970: 23_984_978_378))
         var request: AWSRequest?
         XCTAssertNoThrow(request = try AWSRequest(operation: "test", path: "/", httpMethod: .GET, input: a, configuration: createServiceConfig()))
-        #if os(Linux) && compiler(>=5.5)
+        #if os(Linux) && compiler(>=5.5) && canImport(_Concurrency)
```

Comment:
Does this one need to change? It doesn't look like the code in the block actually relies on concurrency and this is more a 5.5 quirk

Reply:
good catch